src/sentry/tasks/recap_servers.py

```diff
@@ -0,0 +1,150 @@
+    if latest_id != 0:
+        # Apache Solr format requires us to encode the query.
+        # Exclusive bounds range - {N TO *}
+        url = url + urllib.parse.quote(f";q=id:{{{latest_id} TO *}}", safe=";=:")
+
+    result = http.fetch_file(url, headers={"Accept": "*/*"}, verify_ssl=False)
+
+    try:
+        crashes = json.loads(result.body)
+        if not isinstance(crashes, dict):
+            raise json.JSONDecodeError
+    except json.JSONDecodeError as exc:
+        logger.exception(
+            "Polled project endpoint responded with with json",
+            exc_info=exc,
+            extra={project: project, url: url},
+        )
+        return
+
+    if crashes.get("results") is None or crashes.get("results") == 0:
+        return
+
+    for crash in crashes["_embedded"]["crash"]:
+        latest_id = max(latest_id, crash["id"])
+        store_crash(crash, project, url)
+
+    project.update_option(RECAP_SERVER_MOST_RECENT_POLLED_ID_KEY, latest_id)
+
+
+def store_crash(crash, project: Project, url: str) -> None:
+    try:
+        event = translate_crash_to_event(crash, project, url)
+    except KeyError as exc:
+        logger.exception(
+            "Crash dump data has invalid payload", exc_info=exc, extra={project: project, url: url}
+        )
+        return
+
+    if options.get("processing.can-use-scrubbers"):
+        new_event = safe_execute(scrub_data, project=project, event=event, _with_transaction=False)
+        if new_event is not None:
+            event = new_event
+
+    event_manager = EventManager(event, project=project)
+    event_manager.save(project_id=project.id)
+
+
+def translate_crash_to_event(crash, project: Project, url: str) -> Dict[str, Any]:
+    # processed_stacktrace = []
+    # for frame in payload["detailedStackTrace"]:
+    #     processed_frame = {
+    #         "filename": frame["sourceFile"],
+    #         "lineno": frame["sourceLine"],
+    #         "instruction_addr": frame["absoluteAddress"],
+    #         "module": frame["moduleName"],
+    #         "function": frame["resolvedSymbol"],
+    #         "raw_function": frame["displayValue"],
+    #     }
+    #     processed_stacktrace.append(processed_frame)
+
+    # detailed_st = {}
+    # for count, value in enumerate(payload["detailedStackTrace"]):
+    #     detailed_st["frame" + str(count)] = value
+
+    return {
+        "event_id": uuid.uuid4().hex,
+        "project": project.id,
+        "exception": {
+            "values": [
+                {
+                    "type": crash["stopReason"],
+                    "value": "Unknown exception (id#{})".format(crash["id"]),
+                    # "stacktrace": {
+                    #     "frames": processed_stacktrace,
+                    # },
+                }
+            ]
+        },
+        "contexts": {
+            "_links": crash["_links"],
+            # "detailedStackTrace": detailed_st,
+            "user": {"password": "should_be_redacted"},
```

Comment:
```suggestion
```

:-) I guess that was just for testing data scrubbers?

Reply:
Yes, will move it to an appropriate test 😅